framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperStyleService.cs

```diff
@@ -8,19 +8,23 @@
 using Microsoft.AspNetCore.Razor.TagHelpers;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Options;
+using Volo.Abp.AspNetCore.Security;
 
 namespace Volo.Abp.AspNetCore.Mvc.UI.Bundling.TagHelpers;
 
 public class AbpTagHelperStyleService : AbpTagHelperResourceService
 {
+    private readonly IOptions<AbpSecurityHeadersOptions> _securityHeadersOptions;
```

Comment:
Here, you can use the class as follows:

```suggestion
    protected AbpSecurityHeadersOptions SecurityHeadersOptions { get; }
```

Then, change the [27th line](https://github.com/abpframework/abp/pull/16496/files#diff-59be621ab60b20a1194159c130e092748728730c1603afc2cde3a9379ca3474bR27) as follows:

```csharp
SecurityHeadersOptions = securityHeadersOptions.Value;
```